src/lib/assets/yaml.ts

```diff
@@ -6,13 +6,15 @@ import crypto from "crypto";
 import { promises as fs } from "fs";
 import { Assets } from ".";
 import { apiTokenSecret, service, tlsSecret, watcherService } from "./networking";
-import { deployment, moduleSecret, namespace, watcher } from "./pods";
+import { deployment, moduleSecret, namespace, watcher, genEnv } from "./pods";
 import { clusterRole, clusterRoleBinding, serviceAccount, storeRole, storeRoleBinding } from "./rbac";
 import { webhookConfig } from "./webhooks";
-import { genEnv } from "./pods";
-// Helm Chart overrides file (values.yaml) generated from assets
-export async function overridesFile({ hash, name, image, config, apiToken }: Assets, path: string) {
-  const overrides = {
+
+// Generates the overrides object
+export function generateOverrides(assets: Assets, image: string, apiToken: string) {
```

Comment:
```suggestion
export function generateOverrides(assets: Assets, image: string, apiToken: string) {
```

The function name is clear

Reply:
✅ Done